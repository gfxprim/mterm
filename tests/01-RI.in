5 5
a
\eM
