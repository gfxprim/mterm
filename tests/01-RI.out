 -----
|     |
|a    |
|     |
|     |
|     |
 -----
size 5x5 cursor 0x1
